lib/install/validate/validator.go

```diff
@@ -427,11 +423,9 @@ func (v *Validator) network(ctx context.Context, input *data.Data, conf *config.
 	// port forwarding
 	conf.AddNetwork(bridgeNet)
 
-	if v.IsVC() {
-		err := v.checkVDSMembership(ctx, input.BridgeNetworkName)
-		if err != nil {
-			v.NoteIssue(fmt.Errorf("Unable to check hosts in vDS for %q: %s", input.BridgeNetworkName, err))
-		}
+	err = v.checkVDSMembership(ctx, input.BridgeNetworkName)
```

Comment:
https://github.com/vmware/vic/pull/1747/commits/7dd9118932c06e86921cf27470115c6414cf1a2e#diff-a7c93db9877e86593965b23165a2b99fR394
is where the network is checked, so I think you can just use endpointMoref